src/spas/libs/member-management/feature/src/lib/member-management/member-management.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from './user-profile'
+
+@Component({
+  selector: 'jwadmin-member-management',
```

Comment:
remove jwadmin prefix from selector

```typescript
@Component({
  selector: 'member-management',
  templateUrl: './member-management.component.html',
  styleUrls: ['./member-management.component.scss']
})
```

will need to also update `angular.json` to remove the jwadmin prefix to member-management around line 472:

before
```json
    "member-management-feature": {
      "projectType": "library",
      "root": "libs/member-management/feature",
      "sourceRoot": "libs/member-management/feature/src",
      "prefix": "jwadmin",
      "architect": {
```

after
```json
    "member-management-feature": {
      "projectType": "library",
      "root": "libs/member-management/feature",
      "sourceRoot": "libs/member-management/feature/src",
      "prefix": "member-management",
      "architect": {
```